cvat-ui/src/components/create-project-page/create-project-content.tsx

```diff
@@ -2,24 +2,34 @@
 //
 // SPDX-License-Identifier: MIT
 
-import React, {
-    useState, useRef, useEffect, RefObject,
-} from 'react';
-import { useDispatch, useSelector } from 'react-redux';
-import { useHistory } from 'react-router';
-import { Col, Row } from 'antd/lib/grid';
+import React, {RefObject, useEffect, useRef, useState,} from 'react';
+import {useDispatch, useSelector} from 'react-redux';
+import {useHistory} from 'react-router';
+import {Col, Row} from 'antd/lib/grid';
```

Comment:
Please, run linter on all files you have changed. It shows a lot of issues
![Screenshot from 2021-02-16 11-11-55](https://user-images.githubusercontent.com/40690378/108035378-eca29900-7047-11eb-9816-319b4f3a4775.png)